src/components/MainComponent.js

```diff
@@ -31,6 +51,7 @@ export default class MainComponent extends React.Component {
                   <div>
                     { isChromeExtensionBackground() ? <Notifications /> : null }
                     <TopNav />
+                    { isPluginDefined() ? <Route path='/tickets/:ticketId/checklist' component={Checklist} /> : null}
```

Comment:
Seems like it could be useful to have a PluginRoute component that takes like path, component (component name to use) and pluginPath (for the require statement). That could also standardize the user experience when a component isn't available; I'd rather see an error screen saying something like 'Checklist plugin could not be loaded, please contact your system administrator' than just not being able to navigate to a route (which may have been working the previous day prior to some breaking update).